README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+
+## ABOUT ME
+
+###Introduction
+
+My name is Kriti Bhardwaj, a second year student at Harris School of Public Policy. I am pursuing the MS in Environmental Science and Policy. I have 5 years of work experience as an environmental consultant in India.
+
+###Hobbies
+
+In my free time I love to read fiction/mythological fiction, explore architecture and write poetry.
+
+My top three favorite books are:
+
+* **The Fountainhead** by *Ayn Rand*
+* **Kane and Abel** by *Jeffrey Archer*
+* **The Alchemist** by *Paulo Coehlo*
+
+###Extra
+
+As my 30th birthday gift, my friend made a website for me https://kritibhardwaj.com, which I haven't gotten around to editing, so please don't judge! 
+
+![Boston Harbor](Img_me1.jpg)
```

Comment:
I unfortunately wasn't able to view the photo; possibly, this could be because the file wasn't saved in the same location as the markdown file?  (I'm not sure though, but I know that was one issue I had to work through myself!)

Reply:
If you want to include an image on your computer, you have to copy the file into your repository, then stage and commit the `Img_me1.jpg` file as well as the `README.md`, otherwise we will not be able to see it